### ISSUE TITLE
Black Screen

### ISSUE THREAD

**Issue by rinku98** [Bauxitedev/meteorite#6]
I'm trying to run the game on godot 3.1 and the game doesn't show anything, everything is black

**Comment by Bauxitedev** [comment]
Yup, the game doesn't seem to work on Godot 3.1 yet. I'll have to fix it when I have the time.
